index.html

```diff
@@ -7,37 +7,192 @@
+              <div class="multi-post">
+                      <h2>Tonic</h2>
+                  <ul class="frame-2">
+                      <li class="canopy-text"><h3>CANOPY</h3></li>
+                      <li class="role-text"><h3>Back End Dev</h3></li>
+                      <li class="year-text"><h3>2015</h3></li>
+                  </ul>
+              </div>
+              <div class="text-paragraph">
+                  <p>
+                      A daily selection of privately personalized reads; 
+                      no accounts or sign-ups required.
+                  </p>
+              </div> 
+              
+              <ul class="primary-buttons">
+                  <li><button class="button1" type="button">html</button></li>
+                  <li><button class="button2" type="button">css</button></li>
+                  <li><button class="button3" type="button">Javascript</button></li>
+              </ul>
+                      <button class="see-project" type="button">See Project</button>
+          </section>
+        </section>
+        
+        <section class="about-me-section">
+            <section>
+                <div class="about-myself">
+                    <h1>About Myself</h1>
+                </div>                
+                <p class="about-me-text">
+                    I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+                <div class="link-image">
+                    <p class="connect-link">LET’S CONNECT</p>
+                </div>
+                    <ul class="connect-icon">
+                        <li><a href="#"><img src="./image/Linkedinicon.png" alt="" id="image2" /></a></li>
+                        <li><a href="#"><img src="./image/github.png" alt="" /></a></li>
+                        <li><a href="#"><img src="./image/twitter.png" alt="" width="20" height="20" /></a></li>
+                        <li><a href="#"><img src="./image/M.png" alt="" width="20" height="20" /></a></li>
+                        <li><a href="#"><img src="./image/fingers.png" alt="" width="20" height="20" /></a></li>
+                    </ul>     
+                    <button class="get-resume" type="button">Get My Resume</button>   
+                    <div class="language">
+                        <p>Langauges</p>
+                        <a href="#"><img src="./image/union.svg" alt="Union" /></a> 
+                    </div>
+                    <ul class="language-list">
+                        <li><a href="#"><img src="./image/js.png" alt="" /></a></li>
+                        <li><a href="#"><img src="./image/html.png" alt="" /></a></li>
+                        <li><a href="#"><img src="./image/css.png" alt="CSS" /></a></li>

```

Comment:
### Your Project
![languages (copy)](https://user-images.githubusercontent.com/93039359/177992931-39a8b2a7-9b7a-44d3-a916-21ca58698bd9.png)


### Figma Template
![Figma-template](https://user-images.githubusercontent.com/6615394/177891087-161e43fc-43bc-4ea2-a733-a86d9b442329.png)

- [ ] Great you have implemented some of the changes that the previous reviewer required you to do. Please change the background of the **about-me-section** to white to follow the Figma template :+1: